overview/xarray-in-45-min.ipynb

```diff
@@ -36,23 +36,13 @@
     "%config InlineBackend.figure_format='retina'"
```

Comment:
<p>"dict-like" -&gt; "dictionary-like"</p><p> </p><p>We could add text to these links so</p><p><code>[Dataset](https://xarray.pydata.org/en/stable/data-structures.html#dataset)</code></p><p>and <code>[DataArray](<a href="https://xarray.pydata.org/en/stable/data-structures.html#dataarray" target="_blank">https://xarray.pydata.org/en/stable/data-structures.html#dataarray</a>)</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>